packages/astro/test/fixtures/head-injection/src/components/UsesSlotRender.astro

```diff
@@ -0,0 +1,7 @@
+---
+import SlotRenderComponent from "./SlotRenderComponent.astro";
+---
+
+<SlotRenderComponent>
+    <p slot="slot-name">Paragraph.</p>
```

Comment:
I don't see an assertion that this ever ends up in the DOM anywhere. (That may be OK for this; I'm just calling it out in case it's an accidental omission.)

Reply:
That part doesn't matter here. The slots just exist to create the condition where head was inserted improperly. So we only need to test that the links were injected in the head and not the body.